opencti-platform/opencti-front/src/private/components/events/stix_sighting_relationships/StixSightingRelationshipEditionOverview.tsx

```diff
@@ -456,14 +457,16 @@ const StixSightingRelationshipEditionOverviewComponent: FunctionComponent<Omit<S
           )}
         </Formik>
         {typeof handleDelete === 'function' && (
-          <Button
-            variant="contained"
-            onClick={() => handleDelete()}
-            classes={{ root: classes.button }}
-            disabled={inferred}
-          >
-            {t_i18n('Delete')}
-          </Button>
+          <Stack flexDirection="row" justifyContent="flex-end" gap={2}>
+            <Button
```

Comment:
add a rights check before this button

<Security needs={[KNOWLEDGE_KNUPDATE_KNDELETE]}>